examples/flutter_example/pubspec.yaml

```diff
@@ -21,8 +21,9 @@ dependencies:
     sdk: flutter
 
   one_sdk_flutter:
-    git:
-      url: https://github.com/thunderheadone/one-sdk-flutter.git
```

Comment:
is this being removed on purpose?

Reply:
The project can either reference the Thunderhead Plugin locally or from git, and since we were testing this feature I referenced it locally.  I have reverted added a commented another option below if the user would prefer to reference locally.